src/storages/PerpStorage.sol

```diff
@@ -181,6 +187,12 @@ contract PerpStorage is OwnableUpgradeable, ReentrancyGuardUpgradeable, IPerpSto
     }
   }
 
+  function setMovingWindowConfig(uint256 length, uint256 interval) external onlyOwner {
+    emit LogSetMovingWindowConfig(length, interval);
+    movingWindowLength = length;
+    movingWindowInterval = interval;
+  }
+
```

Comment:
If you set a new `movingWindowInterval`, it could instantly change the `epochTimestamp`s that are used in `getEpochVolume()`

For example, if the current interval is 1000, the lookups might be `epochVolumeBuy[idx][1000]`, `epochVolumeBuy[idx][2000]`, `epochVolumeBuy[idx][3000]`. If you change to an interval of 250, the lookups could instantly be `epochVolumeBuy[idx][2500]`, `epochVolumeBuy[idx][2750]`, `epochVolumeBuy[idx][3000]`. In this case the first two mappings were never stored to, so essentially `setMovingWindowConfig()` has zeroed out the epoch volumes in the older epochs

I think this isn't a big deal, since `setMovingWindowConfig()` will rarely be called, and all it does is "reset" the epoch tracking. Do you also think it's okay to just keep this in mind?